docs/2-getting-started/start-free-with-cloud.md

```diff
@@ -4,73 +4,178 @@ title: Start free with Cloud
-- ✨ **You'll know it's working when**: Your PRs appear in a the project
+<!-- insert a video -->
 
-## Step 2: Launch Recce → See Metadata Diffing
+### Required Files
 
-**What you just unlocked**:
+Recce needs `manifest.json` and `catalog.json` from both **base** and **current** environments for comparison.
 
-- ✅ **Lineage visualization** of your models
-- ✅ **Metadata comparison** between changes and production
-- ✅ **Change detection** automatically highlighted
-- ✅ **Column-level impact analysis**
+#### Base Metadata
 
-**You can now**: Explore which models changed and understand downstream impact
+Production environment is commonly used as the baseline, but any environment can serve as the base.
 
-✨ **You'll know it's working when**: You see colored nodes showing changed models in the lineage view
+Choose one method:
 
-## Step 3: Unlock Data Diffing → Real Data Validation
+**Method 1: Generate locally**
 
-- **What you'll unlock**: Compare actual data between development and production
-- **Setup needed**: Configure your data warehouse connection (1 action)
-- **New value**:
-    - ✅ **Value diff** - see row count changes
-    - ✅ **Profile diff** - compare data distributions
-    - ✅ **Schema diff** - spot structure changes
-    - ✅ **Custom query comparisons** - run any SQL side-by-side
+```
+dbt docs generate --target-path target-base --target <your_prod_target>
+```
 
-**You can now**: Validate that your data changes work correctly
 
-✨ **You'll know it's working when**: You can run "Value Diff" and see actual row count comparisons
+**Method 2: dbt Cloud**<br>
+Deploy → Jobs → Production job → Recent run → Download artifacts
 
-## Step 4: Automate Everything → CI/CD Integration
+**Method 3: dbt Docs server**<br>
+Download directly from URLs:
 
-- **What you'll unlock**: Automatic validation on every PR
-- **Setup needed**: [Set up automated workflows](../7-cicd/scenario-ci.md)
-- **Ultimate value**:
-    - ✅ **Automatic PR checks** - validation runs on every push
-    - ✅ **Team workflows** - standardized validation across team
-    - ✅ **PR blocking** - prevent bad changes from merging
+- `<dbt_docs_url>/manifest.json`
+- `<dbt_docs_url>/catalog.json`
 
-## What's Next?
+#### Current Metadata
 
-After launching Recce, explore:
+Use development environment or PR branch as current to compare against the base.
 
-- [View modified](../3-visualized-change/lineage.md)
-- [View downstream impacts](../4-downstream-impacts/impact-radius.md)
-- [Data diffing](../5-data-diffing/query.md)
+Choose one method:
 
-**Questions?** [Join our Discord](../1-whats-recce/community-support.md) - we're here to help!
+**Method 1: Generate locally**
+
+```
+dbt docs generate --target <your_dev_target>
+```
+
+**Method 2: dbt Cloud**<br>
+Deploy → Jobs → CI job → Recent run → Download artifacts
+
+
+### Upload Files
+- Web interface: Click "Update" on the session you want to update in Recce Cloud.
+- CLI command:
+```
```

Comment:
should be 
\`\`\`bash
recce upload-session --session-id <your_session_id>
\`\`\`